manifests/init.pp

```diff
@@ -174,18 +172,6 @@
     group      => $group,
   }
 
-  if $listen == 'public' {
-    file_line { 'puppetboard listen':
-      path    => "${basedir}/puppetboard/dev.py",
-      line    => " app.run('0.0.0.0')",
-      match   => ' app.run\(\'([\d\.]+)\'\)',
-      require => [
-        File["${basedir}/puppetboard"],
-        Python::Virtualenv["${basedir}/virtenv-puppetboard"]
```

Comment:
Also, this resource was removed in #292 so for users of this feature, the module was not working *and broken* since version 7.0.0 of the module.